src/app/functions.js

```diff
@@ -0,0 +1,201 @@
+import i18n from '../i18n'
+
+const sumAll = numbers => {
+  let points = 0
+  numbers.forEach(number => {
+    points = points + number
+  })
+  return points
+}
+
+export const isThreeOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
```

Comment:
The compare function can lead to wrong results on bigger arrays.
`console.log([1, 2, 3, 4, 5, 5, 4, 3, 2, 1, 1, 2, 3, 4, 5].sort((a, b) => a > b));`

`.sort((a,b) => a - b)` solves it